.github/actions/collect_data/src/parsers/tt_torch_model_tests_parser.py

```diff
@@ -84,33 +91,33 @@ def get_pydantic_test(filepath, name, test, default_timestamp=datetime.now()):
 
     tags = None
 
-    return Test(
+    return OpTest(
+        github_job_id=github_job_id,
+        full_test_name=full_test_name,
         test_start_ts=test_start_ts,
         test_end_ts=test_end_ts,
         test_case_name=name,
         filepath=filepath,
-        category="models",
-        group=None,
-        owner=None,
-        frontend="tt-torch",
-        model_name=model_name,
-        op_name=None,
-        framework_op_name=test["torch_name"],
-        op_kind=None,
-        error_message=error_message,
         success=success,
         skipped=skipped,
-        full_test_name=full_test_name,
+        error_message=error_message,
         config=config,
-        tags=tags,
+        frontend=project,
+        model_name=model_name,
+        op_kind="",
+        op_name="",
+        framework_op_name=test["torch_name"],
+        inputs=[],
+        outputs=[],
```

Comment:
`op_kind`, `op_name`, `inputs`, and `outputs` cannot be empty or null string.  We must populate these inputs in order for the DB pipeline to properly ingest the data.  This PR cannot land until these fields are populated.